src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -152,8 +162,13 @@ public Response startMonitor(
         session.save(sessionModel);
         GTFSDB.commitAndCloseSession(session);
 
-        //Extract the Url and gtfsId to start the background process
-        startBackgroundTask(gtfsRtFeed, updateInterval);
+        if(feedSession == null) {
```

Comment:
@barbeau oh ok, I had not seen that. So I guess simply passing the startTime is the only necessary change? (the ones I made in the monitoring.js file)